MUXSDKStats/MUXSDKStats/MUXSDKStats.h

```diff
@@ -158,7 +158,7 @@ FOUNDATION_EXPORT
  @abstract    Updates the monitor for a player to a new AVPlayerLayer.
  @param       player The new AVPlayerLayer to monitor
  @param       name The name of the player instance to update
- @discussion  Use this method to modify the AVPlayerLayer a Mux player monitor is watching. The player monitor must previously have been created via a monitorAVPlayerViewController call.
+ @discussion  Use this method to change which AVPlayerLayer a Mux player monitor is watching. The player monitor must previously have been created via a monitorAVPlayerViewController call.
```

Comment:
Thanks!

one more edit here, I think `created via a monitorAVPlayerViewController call` should be changed to `created via a monitorAVPlayerLayer call`

Reply:
Sure, I've updated that as well.